lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_location.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:app/core/domain/event/entities/event.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/gen/fonts.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+// TODO
+// import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+class GuestEventLocation extends StatefulWidget {
+  const GuestEventLocation({
+    super.key,
+    required this.event,
+  });
+
+  final Event event;
+  // TODO
+  // static const CameraPosition _kGooglePlex = CameraPosition(
```

Comment:
Whether you leave a TODO some more information, or delete uncomment codes. This action keep the project clean and informative